src/lib.rs

```diff
@@ -355,6 +359,34 @@ mod device_context {
     }
 }
 
+mod device_add_result {
+    use super::*;
+    pub fn account_id(d: &DeviceAddResult) -> UserId {
+        d.account_id().clone()
+    }
+    pub fn segment_id(d: &DeviceAddResult) -> usize {
+        d.segment_id()
+    }
+    pub fn device_private_key(d: &DeviceAddResult) -> PrivateKey {
+        d.device_private_key().clone()
+    }
+    pub fn signing_private_key(d: &DeviceAddResult) -> DeviceSigningKeyPair {
+        d.signing_private_key().clone()
+    }
+    pub fn device_id(d: &DeviceAddResult) -> DeviceId {
+        d.device_id().clone()
+    }
+    pub fn name(d: &DeviceAddResult) -> Option<DeviceName> {
+        d.name().cloned()
+    }
+    pub fn created(d: &DeviceAddResult) -> DateTime<Utc> {
+        d.created().clone()
+    }
+    pub fn updated(d: &DeviceAddResult) -> DateTime<Utc> {
+        d.updated().clone()
```

Comment:
Maybe this got missed in the IronOxide PR, but it looks like all the other places that we expose an `updated` TS we call this `last_updated`. Should probably be consistent and do that here too.

Reply:
This is something that we're wildly inconsistent with in ironoxide, and I've been wanting to do a consistency pass to make them all `last_updated` for a while now. 

In ironoxide-java, we're much more consistent and all the functions are `getLastUpdated`. Since the function is aliased in `lib.rs.in`, this `pub fn updated` doesn't actually get exposed out, but I'll change it for consistency.

Reply:
Is ironoxide in a bad place? If you [search for updated](https://docs.rs/ironoxide/0.16.0/ironoxide/?search=updated) on our docs the only values that come back are `last_updated` methods which seems right to me. 🤷‍♂

Reply:
Oh good point. I hadn't realized that there was some sort of internal consistency. It looks like several of the *Result types have their field called `updated` but their function called `last_updated()`. So it looked inconsistent to me reading the code, but at least everything it exposes out is consistent. I updated my ironoxide PR to call the function `last_updated()`.